src/server/Server.php

```diff
@@ -99,6 +102,9 @@ public function __construct(
 		$this->socket->setBlocking(false);
 
 		$this->unconnectedMessageHandler = new UnconnectedMessageHandler($this, $protocolAcceptor);
+
+		// If you don't want to use security on the server, just delete this line.
+		$this->cookieCache = new CookieCache();
```

Comment:
As was previously mentioned. People configuring their servers are not going to go and edit RakLib to delete this. Add a boolean switch to the `Server` constructor.